src/components/RadioGroup.tsx

```diff
@@ -11,23 +11,29 @@ const Wrapper = createStyled('div')
 const Item = createStyled(styled.div`
   display: flex;
   align-items: center;
-  justify-content: space-between;
+  justify-content: space-around !important;
```

Comment:
please avoid `important` in for styleabel elements (move this to the style configuration in the whitelabel app if necessary)